Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/Coupled/Pump.mo

```diff
@@ -0,0 +1,166 @@
+within Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive.Coupled;
+model Pump "Motor coupled chiller"
+  extends Buildings.Fluid.Interfaces.PartialTwoPort(
+  port_a(p(start=Medium.p_default)),
+  port_b(p(start=Medium.p_default)));
+  extends Buildings.Electrical.Interfaces.PartialOnePort(
+    redeclare package PhaseSystem =
+        Buildings.Electrical.PhaseSystems.OnePhase,
+    redeclare replaceable Interfaces.Terminal_n terminal);
+
+  parameter Boolean addPowerToMedium=true
+    "Set to false to avoid any power (=heat and flow work) being added to medium (may give simpler equations)";
```

Comment:
make the comment more clearer.

Reply:
Thank you for your suggestion! I have revised the comments in related models. Same as mentioned before, originally I just referred to the pump model in MBL (https://github.com/lbl-srg/modelica-buildings/blob/8ab33d18a40dec150af2d10335f31d7cd002db12/Buildings/Fluid/Movers/BaseClasses/PartialFlowMachine.mo#L41). However as this is an extension of original model, it would be better to use clear comment to improve readability of models, I have revised those comments accordingly.